### PR TITLE
feat: Add rrweb to sentry staff sessions

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
Aside this likely records for transactions which isn’t anticipated but will be... interesting (and useful). Office internet might go boom but it already is awful.

File: src/sentry/static/sentry/app/bootstrap.jsx [hunk 1]
```diff
@@ -24,6 +25,27 @@ import Main from 'app/main';
...
+    new ExtraErrorData({
+      // 6 is arbitrary, seems like a nice number
+      depth: 6,
+    }),
+    new Integrations.Tracing({
+      tracingOrigins: ['localhost', 'sentry.io', /^\//],
+      tracesSampleRate,
+    }),
+  ];
+  if (window.__SENTRY__USER && window.__SENTRY__USER.isStaff) {
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/bootstrap.jsx]: Isn't this in `initialData`?

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/bootstrap.jsx]: nvm, I see it below too.

**Review by billyvg — APPROVED** [review]
¯\_(ツ)_/¯

**Pull request merged**